lib/ckb/wallet.rb

```diff
@@ -70,9 +71,19 @@ def get_balance
     # @param data [String ] "0x..."
     # @param key [CKB::Key | String] Key or private key hex string
     # @param fee [Integer] transaction fee, in shannon
-    def generate_tx(target_address, capacity, data = "0x", key: nil, fee: 0, use_dep_group: true)
+    def generate_tx(target_address, capacity, data = "0x", key: nil, fee_rate: 0, use_dep_group: true)
```

Comment:
I am not sure if using fee_rate as default is a good choice. Let's take a look at other people's opinions.